float32 roll
float32 pitch
float32 heading
float32 speed